_data/finance.yml

```
@@ -84,6 +84,12 @@ websites:
     facebook: ditech
     img: ditech.png
 
+  - name: Equifax
+    url: https://www.equifax.com/
+    twitter: EquifaxInsights
```

```suggestion
    twitter: AskEquifax
```
I'd suggest switching to their consumer customer support Twitter username.